### PR TITLE
Fix grammar in README.md and DEVELOPER.md

### PR DESCRIPTION
**Pull request by cgrigel**
Added new badges, fixed grammary in README.md and DEVELOPER.md #hacktoberfest

### CODE REVIEW

**Comment by cgrigel** [comment]
Added new badges, fixed grammary in README.md and DEVELOPER.md #hacktoberfest

File: README.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 [![GitHub tag (latest SemVer pre-release)](https://img.shields.io/github/tag-pre/jenkinsci/jenkinsfile-runner?label=changelog)](https://github.com/jenkinsci/jenkinsfile-runner/releases/latest)
 [![Docker Pulls](https://img.shields.io/docker/pulls/jenkins4eval/jenkinsfile-runner?label=docker%20pulls%20%28vanilla%29)](https://hub.docker.com/r/jenkins4eval/jenkinsfile-runner)
 [![GitHub contributors](https://img.shields.io/github/contributors/jenkinsci/jenkinsfile-runner)](https://github.com/jenkinsci/jenkinsfile-runner/graphs/contributors)
+[![GitHub issues](https://img.shields.io/github/issues/jenkinsci/jenkinsfile-runner)](https://github.com/jenkinsci/jenkinsfile-runner/issues)
+[![GitHub forks](https://img.shields.io/github/forks/jenkinsci/jenkinsfile-runner)](https://github.com/jenkinsci/jenkinsfile-runner/network)
+[![GitHub stars](https://img.shields.io/github/stars/jenkinsci/jenkinsfile-runner)](https://github.com/jenkinsci/jenkinsfile-runner/stargazers)
+
+
```
Comment (oleg-nenashev) [inline README.md]: I suggest reverting it. There badges are already visible on GitHUb

**Comment by cgrigel** [comment]
Dear @oleg-nenashev,
Yes. sure I understand your point concerning the needs of availability of these badges.

**Review by oleg-nenashev — COMMENTED** [review]
@cgrigel could you please remove badges so that I merge the rest of the PR?

**Comment by cgrigel** [comment]
Dear @oleg-nenashev,
Sorry for the radio silence, I was removed unnecessary badges.
Thank you for your collaboration.

**Comment by oleg-nenashev** [comment]
Thanks!

**Pull request merged**